src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
```

Comment:
Are you sure? I don't see that happening for me... All these 2 lines of code are doing first formatting the date passed in to be like `YYYY-MM-DDThh:mm:ssTZD` to guarantee all dates are in the same format. Then I'm just looking for the index of the character `T` in that string, which should always return `10`.